includes/env.h

```diff
@@ -0,0 +1,33 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   env.h                                              :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: takkatao <takkatao@student.42tokyo.jp>     +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/02/23 08:24:54 by takkatao          #+#    #+#             */
+/*   Updated: 2022/02/25 14:19:11 by takkatao         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef ENV_H
+# define ENV_H
+
+# include "define.h"
+# include "xlibft.h"
+
+typedef struct s_keyvalue
+{
+	char			*key;
+	char			*value;
+}	t_keyvalue;
+
+void	init_env(void);
+void	free_keyvalue(void *kv);
+t_list	**gen_env(void);
+char	*get_env(char *name);
+void	set_env(char *name, char *value);
+void	remove_env(char *name);
+void	remove_all_env(void);
```

Comment:
remove_envよりunsetの方が良いかも。builtinのコマンド名もunsetですし。

```suggestion
void	unset_env(char *key);
void	remove_env(void);
```